test/resources/sampleContainer/example1.ttl

```diff
@@ -2,6 +2,8 @@
 @prefix dc: <http://purl.org/dc/elements/1.1/> .
 @prefix ex: <http://example.org/stuff/1.0/> .
 
+</sampleContainer/example*> a :glob.
```

Comment:
Added to address @dan-f's comment; tests the `requestUri` being used as a serialization base for globs. (per @timbl's instructions)